game/scripts/npc/abilities/ember_spirit_sleight_of_fist.txt

```diff
@@ -38,7 +38,7 @@
       "02"
       {
         "var_type"                                        "FIELD_INTEGER"
-        "bonus_hero_damage"                               "40 80 120 160 280 560"
+        "bonus_hero_damage"                               "40 80 120 160 440 720"
```

Comment:
how about 240/640

Reply:
you could make the radius bigger at OAA levels... like 550/575/600

Reply:
I buffed this last month (by 10 though) because the talent is buffed, but it can be buffed more I guess. Your numbers are just weird. It should be a multiplier of 160. 90% of non-ultimate spells at OAA levels have scaling like this:
 lvl5_value = lvl4_value x 2; lvl6_value = lvl4_value x 4 or lvl4_value x 3; Optimal values for Sleight of Fist at OAA levels would be: 320/640 and if you are not happy with these values, buff the Sleight of Fist bonus damage talent.

Reply:
> 
> 
> I buffed magic Ember last month. So you shouldn't buff it more. Sleight of Fist damage buffs are fine but numbers are ugly. Talent changes are a little bit questionable. Having 75 damage at level 10 is a huge powerspike.

Its nice that you buffed this hero, but he needs more. This hero is so bad its unplayable. Post aghs this hero has only really been viable in pubs, now that all the items and creeps ember abused have been nerfed he needs buffs. This hero was only ever good if your double the enemy teams level.